ledger/simulation/simulator_test.go

```diff
@@ -152,6 +152,7 @@ int 1`,
 		mocktracer.AfterOpcode(false),
 		mocktracer.AfterProgram(logic.ModeSig, false),
 		// Txn evaluation
+		mocktracer.BeforeBlock(block.Block().Round()),
```

Comment:
This order was surprising to me, but because LogicSig verification happens before the Txns in the block get evaluated the `BeforeBlock` call will happen after LogicSig verification.